xbmc/interfaces/legacy/ModuleXbmc.cpp

```diff
@@ -380,6 +380,8 @@ namespace XBMCAddon
     String translatePath(const String& path)
     {
       XBMC_TRACE;
+      CLog::Log(LOGWARNING, "xbmc.translatePath is deprecated and might be removed in future kodi "
+                            "versions. Please use xbmcvfs.translatePath instead.");
```

Comment:
Has looked over the code in `xbmc/interfaces/legacy` is there not easy possible to get related addon id?

Think would be good to have on log messages like them the related addon id, otherwise hard to see which one it have.

Reply:
In python we get it through a call to xbmcaddon.Addon().getAddonInfo("id"). I'll try to find out if it's easy enough to get the id on the log line

Reply:
Ok took a look into it. It is not that easy because xbmc.translatePath might not only be called from addons (you could call it for example from autoexec.py).
I suggest to keep it as is.

Reply:
Yes, is OK then. Thought also the same, but was not sure.